travel/src/main/resources/db/migration/V3__dbscheme.sql

```diff
@@ -0,0 +1,88 @@
+CREATE TABLE IF NOT EXISTS `exchange_rates` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `code` VARCHAR(20) DEFAULT NULL,
+    `amount_in_rsd` FLOAT UNSIGNED DEFAULT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `UK_code` (`code`)
+    );
+
+CREATE TABLE IF NOT EXISTS `travel_expense_forms` (
+    `id` VARCHAR(36) NOT NULL,
+
+    `departure_time` TIME NOT NULL,
```

Comment:
maybe use timestamp here, so it is more standardized?

Reply:
I totally agree! I will change that throughout the whole scheme.